css/templates/lune/pages/_landing.css

```diff
@@ -127,8 +132,18 @@
       height: calc(var(--lune-landing-page-height) / 0.99);
     }
 
-    .lune-landing-wrapper {
+    .lune-landing-content {
```

Comment:
Setting `position: fixed` on `.lune-landing-wrapper` was causing a few bugs related to pointer events. To avoid these bugs, we have to set `position: fixed` on `.lune-landing-content` and `.lune-component-scroll-indicator` instead, and thus the only way to center them in the page is by using `transform`.

This may not be very flexible, but it's the only way of achieving the design, which is very bespoke for this page.